### PR TITLE
[RM-86764] Added nvdDatafeedUrl and analyzersKnownExploitedURL optional properties

### PR DESCRIPTION
**Pull request by Nikita-Smirnov-Exactpro**
…al properties

### CODE REVIEW

**Review by OptimumCode — APPROVED** [review]
Can we add an integration test to see if the default settings still work (no URLs were provided)?

File: plugin/src/test/kotlin/com/exactpro/th2/gradle/BaseTh2PluginTest.kt [hunk 1]
```diff
@@ -29,17 +30,21 @@ import org.junit.jupiter.api.assertThrows
...
 
 @TestInstance(PER_CLASS)
 internal class BaseTh2PluginTest {
     @ParameterizedTest(name = "when {0} plugin applied")
     @MethodSource("requiredPlugins")
     fun `applies required plugins`(javaPlugin: String) {
-        val project =
-            ProjectBuilder.builder()
-                .build()
+        val project = ProjectBuilder.builder().build()
```
Comment (OptimumCode) [inline plugin/src/test/kotlin/com/exactpro/th2/gradle/BaseTh2PluginTest.kt:47]: question: Why do you need to change the formatting in places where you don't change the code?

**Review by OptimumCode — APPROVED** [review]

File: plugin/src/test/kotlin/com/exactpro/th2/gradle/BaseTh2PluginTest.kt (hunk 1 above)
Comment (Nikita-Smirnov-Exactpro) [reply plugin/src/test/kotlin/com/exactpro/th2/gradle/BaseTh2PluginTest.kt:47]: I saw warning in idea

**Comment by Nikita-Smirnov-Exactpro** [comment]
> Can we add an integration test to see if the default settings still work (no URLs were provided)?

If we add test without nvdApiKey, the test takes a long time.
If we pass nvdApiKey via system property (-DnvdApiKey=...), the obvious value is printed into gradle info log

**Pull request merged**